Makefile

```diff
@@ -211,10 +211,18 @@ build-all-in-one-linux: build-ui
 build-all-in-one: elasticsearch-mappings
 	CGO_ENABLED=0 installsuffix=cgo go build -tags ui -o ./cmd/all-in-one/all-in-one-$(GOOS) $(BUILD_INFO) ./cmd/all-in-one/main.go
 
+.PHONY: build-all-in-one-without-ui
+build-all-in-one-without-ui: elasticsearch-mappings
+	CGO_ENABLED=0 installsuffix=cgo go build -o ./cmd/all-in-one/all-in-one-$(GOOS) $(BUILD_INFO) ./cmd/all-in-one/main.go
```

Comment:
Just realized that this target is exactly the same as the `build-all-in-one`. They could perhaps share the same command, just changing the value for the "tags" flag. Like:

    CGO_ENABLED=0 installsuffix=cgo go build -o ./cmd/query/query-linux -tags "${TAGS}" ./cmd/query/main.go

One would set `TAGS` to `ui`, one would keep it empty. This is not blocking the PR, but would be a nice thing to have.

Reply:
not just here, but a lot of new `*-without-ui` targets added below, which are identical to existing ones aside from `-tags`. I agree, it's better to control that aspect with a variable rather than replicating targets.